docs/src/references/c8y-configuration-management.md

```diff
@@ -162,4 +164,104 @@ Note that:
   for instance `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`.
 * Since the type of configuration file is used as an MQTT topic name, the characters `#` and `+` cannot be used in a type name.
   If such a character is used in a type name (or in the path of a configuration file without explicit type),
-  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
\ No newline at end of file
+  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
+
+## Configuration files for child devices
+
+To manage configuration files for child-devices the `c8y_configuration_plugin` supports two aspects:
+1) Allowing to associate a configuration file with a cloud's child-device twin
+2) Allowing to consume/provide a configuration file from/to an external device via network
+
+**Details to Aspect 1: Associating to cloud's child-device twin**
```

Comment:
The term _child device_ is a bit weak, cause it could mean the twin in the cloud, or some representation within thin-edge, or an external device that represents the child-device. Therefor I have tried to avoid using the term _child device_, but instead used _external device_ or _child device twin_.

I am currently about to create a unique definition of terminology as part of an overall documentation for the child-device topic. I will link that document here in that plugin spec.